contracts/base/BaseAccount.sol

```diff
@@ -82,14 +82,13 @@ contract BaseAccount is IBaseAccount {
         address entryPointAddress = _ENTRYPOINT;
         /// @solidity memory-safe-assembly
```

Comment:
The following block is no longer memory safe, since it doesn't update the free memory pointer.

Reply:
good catch